kcidb/monitor/output.py

```diff
@@ -174,18 +177,22 @@ def render(self):
             self.obj.get_type().name: self.obj,
         }
         subject = TEMPLATE_ENV.from_string(self.message.subject).render(ctx)
-        subject_extra_bytes = \
-            len(subject.encode()) - NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN
-        assert subject_extra_bytes <= 0, \
-            f"Subject is {subject_extra_bytes} bytes too long"
+        subject_extra_characters = \
+            len(subject) - NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN
+        if subject_extra_characters > 0:
+            subject = subject[:NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN]
+            LOGGER.warning("Subject is %s characters too long",
```

Comment:
Could you note what's being done about the subject, in the log message? Like this:
```suggestion
            LOGGER.warning("Subject is %s characters too long, truncated",
```
And same for the body message?